### DISCUSSION TITLE
Change color of tabs and window frame

### DISCUSSION THREAD

**Discussion by Croqueta0** [sumatrapdfreader/sumatrapdf#4080] [Ideas]
The color of the tabs and window frame should be customizable. When one sets darker colors for the pdf, the white of the tabs is too distracting (sure, I can use f11, but one wants to be able to use the tabs anyway). 

I think the colors of the menu, scroll bars, command line and so on should also be adjustable, for the same reason. 

I enjoy PDF sumatra a lot and I think this little feature would make it more enjoyable. :)

**Comment by GitHubRulesOK** [comment]
there has always been windows frame theming of the windows system controlled colours since turn of the century XP however Microsoft keep moving those goalposts and SumatraPDF has other adjustments to function internally so simply use a third party app for scrollbars and title bar (there are many examples throughout the "issues") especially there seems to even less ability for windows blinds to work in recently changed Win11.

for tabs there are multiple colors of black from the lightest of white to the max that Microsoft allow but collateral damage is black icons on dark blackground look like s##tware co-UP this is Windows 10 `system colors` with recent light / dark / darker active
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/9b4c2413-9b90-4eb8-8a28-1a6e76530e4f)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/ed273e2d-c93d-4c0e-a383-464767440fb5)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/8d75aa0c-9de5-406d-978c-2aeb2e303ff4)

**Reply by Croqueta0** [reply]
Oops, I didn't check for updates. The dark themes of Sumatra seem to be good enough for what I wanted, thanks!

**Comment by Av1ators** [comment]
the icons in night/black mode are grainy and ruined. is a fix possible?

**Comment by PhilBaz** [comment]
Is there a way to change the color of the active tab? I'm using darker and it goes to dark black, that doesn't stand out to me. Would love to have a bright accent color here.
